scripts/xtalk_dayenu_filter_run_baseline_parallelized.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2020 the HERA Project
+# Licensed under the MIT License
+
+"Command-line drive script for hera_cal.xtalk_filter. Only performs DAYENU Filtering"
```

Comment:
This should be a docstring with `"""` on each side rather than just a plain string.

Reply:
i've triple stringed it.